### PR TITLE
write wrong

### PR DESCRIPTION
**Pull request by huangchaoshun**

### CODE REVIEW

**Comment by huangchaoshun** [comment]
write wrong

File: part1 [hunk 1]
```diff
@@ -1,2 +1,2 @@
 This is some test about part1
-add some thing
+add some thing, i wirte wrong.
```
Comment (hcsxiaohan) [inline part1]: look your wirte

Comment (hcsxiaohan) [reply part1]: look Again

Comment (huangchaoshun) [reply part1]: oh,shit

**Pull request closed without merging**